### ISSUE TITLE
Title is computed incorrectly for work week (no weekends)

### ISSUE THREAD

**Issue by gas12n** [fullcalendar/fullcalendar#2884]
I have a custom view that is defined as:

```
agendaNoWeekend: 
{
  type: "agendaWeek",
  buttonText: "Work week",
  weekends: false
}
```

The function `computeTitle` uses `intervalStart`  and `intervalEnd` which include weekends. My undestanding is that `start` and `end` have hidden days skipped. By using `start` and `end` the title will reflect the period of time the calendar is actually showing (e.g. in this case it won't show the weekend). 

A byproduct of using `start` and `end` is that in month view for October it would show something like  September - November because that's the actual period the calendar is showing (last few days of September and first few of October).

This change seems to work for me:

```
computeTitle: function () {

            var startDate, endDate;

            if (this.intervalUnit == 'month') {
                startDate = this.intervalStart;
                endDate = this.intervalEnd;
            }

            else {
                startDate = this.start;
                endDate = this.end;
            }

            return this.formatRange(
            { start: startDate, end: endDate },
                this.opt('titleFormat') || this.computeTitleFormat(),
                this.opt('titleRangeSeparator')
            );
        }
```

Cheers

**Comment by arshaw** [comment]
hi @gas12n thanks for the report, but would you be able to post a demoable recreation as well? http://fullcalendar.io/wiki/Reporting-Bugs/

would really help. thanks

**Comment by gas12n** [comment]
Here is the demo: http://jsbin.com/bukuqupico/edit?js,output

Switch between week and work week view. Notice how the title in the center does not change even though the range shown has.

**Comment by arshaw** [comment]
ahh i see. thanks

**Comment by JsGuru30** [comment]
Sorry to bother but has this issue been resolved this? I am experiencing same problem and would like to know how to fix it.
Thanks.

**Comment by arshaw** [comment]
Fixed in [v3.2.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.2.0)
